store/index.js

```diff
@@ -38,7 +38,14 @@ export const mutations = {
       return response.data.body.filter((body) => body.primary.type === type)
     }
 
-    state.organizationNavItems = navItems(navigations, 'organizations-nav')
+    if (process.env.isPixSite) {
+      state.organizationNavItems = navItems(navigations, 'organizations-nav')
+    } else {
+      state.organizationNavItems = navItems(
+        navigations,
+        'pix-pro-organizations-nav'
+      )
+    }
```

Comment:
Petit suggestion de refacto : 

```suggestion
    state.organizationNavItems = navItems(navigations, process.env.isPixSite ? 'organizations-nav' : 'pix-pro-organizations-nav')
```